content/user-guide/catalog-projects.md

```diff
@@ -17,7 +17,7 @@ This view shows the content of a catalog project.
 
 ## Collaboration / Access Rights
 
-Other than [projects](../projects), catalog projects do not allow you to manage access rights. All [members of your organization](../settings#managing-members-admin-only) automatically have _read-only_ access to your catalog project, meaning that they can see your templates, but not edit them.
+Catalog projects are [projects created via integrations](../projects/#projects-created-via-integrations) and do not allow you to manage access rights like [projects created from Cawemo](../projects/#projects-created-from-cawemo) do.
```

Comment:
This might be confusing for users who aren't familiar with Cawemo too much as catalogs are created manually by users and not via integrations.

Reply:
Good point, I will try to rephrase to clarify that.

Reply:
Maybe:

```suggestion
## Collaboration / Access Rights

Catalog projects are [projects used by integrations](../projects/#projects-used-by-integrations) and do not allow you to manage access rights like [projects created from Cawemo](../projects/#projects-created-from-cawemo) do.

You can create Catalog projects and as the owner of a catalog project, only you have the right to create, edit, and delete templates.
```

Is this clearer?

Reply:
Yep, the first paragraph is clearer now. I'd suggest the following for the second paragraph:

```
Everybody can create Catalog projects but only the owner of a catalog project has the right to create, edit, and delete templates.
```

Reply:
Nice suggestion! 💯